daemon/logger/journald/journald.go

```diff
@@ -85,7 +86,11 @@ func New(ctx logger.Context) (logger.Logger, error) {
 	for k, v := range extraAttrs {
 		vars[k] = v
 	}
-	return &journald{vars: vars, readers: readerList{readers: make(map[*logger.LogWatcher]*logger.LogWatcher)}}, nil
+	return &journald{
+		vars:           vars,
+		gatewayAddress: ctx.Config["journal-gateway-address"],
```

Comment:
`gateway-address` is sufficient, since these options are only passed to `journald` log driver.